libflux/src/flux/semantic/nodes.rs

```diff
@@ -128,8 +128,8 @@ pub enum Assignment {
 impl Assignment {
     fn apply(self, sub: &Substitution) -> Self {
         match self {
-            Assignment::Variable(ass) => Assignment::Variable(ass.apply(&sub)),
-            Assignment::Member(ass) => Assignment::Member(ass.apply(&sub)),
+            Assignment::Variable(assign) => Assignment::Variable(assign.apply(&sub)),
+            Assignment::Member(assign) => Assignment::Member(assign.apply(&sub)),
```

Comment:
This change made me smirk. It's appreciated.

Reply:
Yeah, I had to read that file too many times to leave it as is.